.github/SECURITY.md

```diff
@@ -8,12 +8,12 @@ updates:
   - The most recent release, and the release prior to that.
   - Active LTS releases.
 
-At this time, with the latest release of v3.2.0, the supported
+At this time, with the latest release of v3.3, the supported
 versions are:
 
-  - v2.7.0: Current LTS
-  - v3.1.0: Prior release
-  - v3.2.0: Current release
+  - v2.7: Current LTS
```

Comment:
Should this also say what the current version is specifically for LTS e.g. 2.7.4? Since there are certain third party tools of which we shall not speak... that have older versions of 2.7 with known bugs

Reply:
The rationale for removing the "patch level" is described in the commit message:

```
1. It currently does not convey any meaningful information (simply set
   to "0") and is really unnecessary because the support status is
   defined at the "minor" version level.
    
2. Updating the security policy document whenever there is a new "patch
   release" is arguably too noisy and does not add much value.
```

I am also ok with specifying the latest patch level here; but, it will be quite cumbersome to maintain (and release managers may often forget to update this because we do not strictly follow a checklist for patch releases).

Reply:
For the maintenance burden, fine to keep it without patch level.